### DISCUSSION TITLE
Unable to use toMail from my notification

### DISCUSSION THREAD

**Discussion by abhiburk** [ylsideas/subscribable-notifications#51] [Q&A]
Hey please guide me for this
I am using default notification mail function as given below but it never gets called now Please help me out of this. I have my own content in rate_your_experience mail blade file but that is not being use.
Check my screenshot this is what my email is looking in Inbox which should not look like. `rate_your_experience` blade file has my custom content,

![Screenshot 2021-07-08 at 20-41-29 Mailtrap - Safe Email Testing](https://user-images.githubusercontent.com/21901478/124946740-e4fe8380-e02c-11eb-831e-a86cbc5e8423.png)

It seems that my markdown template is not in use when triggering email using below syntax.

```
public function toMail($notifiable)
{
    return (new MailMessage)
            ->subject("Rate Your Experience")
            ->markdown(
                'emails.rate_your_experience',
                [
                    'source' => $this->source,
                    'user' => $notifiable
                ]
            );
}
```

**Comment by Oreolek** [comment]
I also have this problem after installing the package - all my Markdown emails are suddenly empty default templates.

UPD: yup, the reason is in the `SubscriberMailChannel::buildView` function. It bails if the message has a `view` property but ignores the `markdown` one. Oh well, guess this package is incompatible.
